.travis.yml

```diff
@@ -110,15 +110,6 @@ jobs:
       script:
         - ./scripts/if_changed.sh ./scripts/build.sh $PROJECT $PLATFORM $METHOD
 
-    - stage: test
```

Comment:
Is this causing any actual problems yet? If not I’d prefer to leave it.

Even though we don’t actually support Xcode 8 it represents an older implementation of the C++11 standard and helps us avoid using later language features that are implicitly implemented by later compilers.